README.md

```diff
@@ -6,7 +6,7 @@
     height="250px"
   />
 </p>
-<h1 align="center"><b>Aries JavaScript Documentation</b></h1>
+<h1 align="center"><b>Credo Documentation</b></h1>
 <p align="center">
   <a
     href="https://raw.githubusercontent.com/hyperledger/aries-framework-javascript/main/LICENSE"
```

Comment:
Outdated link (i think we should do a find al on aries-framework-javascript and hyperledger as well